recipes/transcov/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "transcov" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 42d988e377fb42862938abe14ee0adb50d54d219c16ea61e351a8908788a60d5
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - transcov = transcov.cli:cli
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - attrs
+    - click >=6.0
+    - numpy
+    - pip
+    - pysam
+    - python >=3.6
```

Comment:
Do you need all these requirements in host? python and pip should be enough I'd have thought.

Reply:
No, I think they can be removed.